R/tostring.R

```diff
@@ -391,7 +391,7 @@ do_cell_fnotes_wrap <- function(mat, widths, max_width, tf_wrap, fontspec, expan
   correct_indentation <- vapply(seq_along(mf_lgrp), function(xx) {
     grouping <- mf_lgrp[xx]
     if (nzchar(real_indent[grouping])) {
-      has_correct_indentation <- stringi::stri_detect(
+      has_correct_indentation <- stri_detect(
```

Comment:
why to remove prefixes? It feels like a step back. Code with prefixes are safer against the namespace conflicts. I'm not asking for adding prefixes everywhere (it's going to be a lot) but not to remove existing ones :)

Reply:
that's correct move
this does not imply we need to remove prefixes. Packages from Imports can still have prefixes.

Reply:
```suggestion
      has_correct_indentation <- stringi::stri_detect(
```